### PR TITLE
add functions to read instrument info from DL1 file

### PR DESCRIPTION
**Pull request by vuillaut**
Necessary as long there is no DL1 event source

### CODE REVIEW

File: lstchain/io/io.py [hunk 1]
```diff
@@ -443,6 +445,239 @@ def read_array_info(filename):
     return array_info
 
 
+def read_optic(filename, telescope_name):
```
Comment (moralejo) [inline lstchain/io/io.py:448]: Function name is not so nice, and a bit too close to the other one. Any better option?
e.g. read_single_optics    (and keep read_optics as it is)

Comment (moralejo) [reply lstchain/io/io.py:448]: And thanks a lot for this! Will try it.

Comment (vuillaut) [reply lstchain/io/io.py:448]: Good point, I will rename this one and similar with `_single_`

**Comment by vuillaut** [comment]
I have absolutely no idea why the test is failing on something completely unrelated and that is working fine otherwise: `CameraDisplay(geom, image)`
`astropy` version ?

**Comment by rlopezcoto** [comment]
> I have absolutely no idea why the test is failing on something completely unrelated and that is working fine otherwise: CameraDisplay(geom, image) astropy version ?

I'm getting those tests pass in my local version of this PR, are they also passing in your computer?

**Comment by vuillaut** [comment]
> > I have absolutely no idea why the test is failing on something completely unrelated and that is working fine otherwise: CameraDisplay(geom, image) astropy version ?
> 
> I'm getting those tests pass in my local version of this PR, are they also passing in your computer?

yes they are.
It must be an install problem but it does not seem to replicate on other PRs... 🙄

**Comment by rlopezcoto** [comment]
then, let's move on...

**Pull request merged**